modules/dag-shared/src/main/scala/org/tessellation/dag/transaction/filter/Consecutive.scala

```diff
@@ -2,33 +2,56 @@ package org.tessellation.dag.transaction.filter
 
 import cats.data.NonEmptyList
 import cats.effect.Async
+import cats.syntax.flatMap._
 import cats.syntax.functor._
 import cats.syntax.list._
 import cats.syntax.traverse._
-import cats.{Applicative, Order}
+import cats.{Applicative, Id, Order}
 
 import scala.annotation.tailrec
 
 import org.tessellation.kryo.KryoSerializer
 import org.tessellation.schema.transaction.{Transaction, TransactionReference}
 import org.tessellation.security.Hashed
+import org.tessellation.security.signature.Signed
 
 object Consecutive {
+  val hashedTxOrder: Order[Hashed[Transaction]] =
+    Order.whenEqual(Order.by(-_.fee.value.value), Order[Hashed[Transaction]])
```

Comment:
You can achieve more DRY solution here.
`Hashed[Transaction]` has already `Signed[Transaction]` inside so you can create `Order[Hashed[Transaction]]` by using contravariant functor:

```scala
import cats.syntax.contravariant._

val hashedTxOrder: Order[Hashed[Transaction]] = signedTxOrder.contramap(_.signed)
```

Reply:
Is it necessary though? `Hashed` has auto unwrap. This should work

```scala
val hashedTxOrder: Order[Hashed[Transaction]] = Order.reverse(Order.by(_.fee))
```

Reply:
We not only need it to sort by fee but also to be deterministic when different nodes run the same selection process. So I need to order by fee and then by everything else...